variables.tf

```diff
@@ -123,3 +123,13 @@ variable "kubeconfig_name" {
   description = "Override the default name used for items kubeconfig."
   default     = ""
 }
+
+variable "aws_eks_cluster_create_timeout" {
```

Comment:
For consistency with other `cluster_*` related variables and readability (e.g. `cluster_version`, should these be named `cluster_create_timeout` and `cluster_delete_timeout`?

Reply:
Sure, I'll change it